buildchain/buildchain/image.py

```diff
@@ -297,7 +297,7 @@ def show() -> str:
     ),
     targets.LocalImage(
         name='metalk8s-ui',
-        version='0.2',
+        version='2.0',
```

Comment:
You should use `constants.VERSION` here but it's merged, I will do the changes in my PR about using SHORT_VERSION instead of VERSION #1251